### PR TITLE
Consistent splash colors and logos

### PR DESCRIPTION
**Pull request by ecotopian**

### CODE REVIEW

File: Client/Ecosia/UI/Ecosia.xcassets/ecosiaLogoLaunch.imageset/Contents.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
       "scale" : "2x"
     },
     {
-      "filename" : "mainInHouseUseEcosiaLogoColor@3x.png",
+      "filename" : "logoMainEcosiaLight@3x.png",
```
Comment (vauxhall) [inline Client/Ecosia/UI/Ecosia.xcassets/ecosiaLogoLaunch.imageset/Contents.json:36]: Why using pdf for light but png for dark?

File: Client/Info.plist [hunk 2]
```diff
@@ -148,8 +148,13 @@
 	</array>
 	<key>UIFileSharingEnabled</key>
 	<true/>
-	<key>UILaunchStoryboardName</key>
-	<string>LaunchScreen</string>
+	<key>UILaunchScreen</key>
+	<dict>
+		<key>UIColorName</key>
```
Comment (vauxhall) [inline Client/Info.plist:153]: Nice

**Review by vauxhall — APPROVED** [review]
💯

File: Client/Info.plist (hunk 2 above)
Comment (ecotopian) [reply Client/Info.plist:153]: yeah I haven't know this API before today: here's the article explaining it: https://www.avanderlee.com/xcode/launch-screen/

Comment (vauxhall) [reply Client/Info.plist:153]: Interesting, is it compatible with older iOS versions?

Comment (ecotopian) [reply Client/Info.plist:153]: no 😭 - just checked.. seems to only work for iOS14 and up 😢 will try to find a workaround

**Pull request merged**